nh.c

```diff
@@ -64,6 +64,56 @@ __attribute__((visibility("hidden"))) void nh_failsafe_destroy(nh_failsafe_t *fs
 // used afterwards.
 __attribute__((visibility("hidden"))) void nh_failsafe_uninstall(nh_failsafe_t *fs);
 
+// The following blacklists help prevent any whoopsies when deleting files during uninstall
+
+// These files are important for recovery purposes.
+static const char *delete_file_blacklist[] = {
```

Comment:
Yeah, I don't have one of those, only an H2O and a Forma :p.

Reply:
Then I should probably do the testing :D.

Reply:
Sorry ^^

Reply:
It's OK (I leave the back open anyways, as I use that one purely for testing, so it's not really hard to recover if it goes wrong).

Reply:
Yeah, If I ever have to open my H2O, I'm planning to take a similar approach as @froschtschutz: cut out a hole right above the SD card slot with an exacto knife or something, because I *know* that I'm terrible at opening clipped/glued plastic cases  ;).